index.js

```diff
@@ -74,8 +108,34 @@ function makeGrid(){
     }
     gridWrapper.appendChild(fragment);
     Wrapper.appendChild(gridWrapper);
+    borderGrid();
+    setGridDivWidth();
 };
 
+function setGridDivWidth(){
+    var div = document.querySelector("#Wrapper > div");
+    div.style.width = (gridEdgeLenght + "px");
+}
+
+function borderGrid(){
+    var borderTop = document.querySelectorAll(".row-1 .cell");
```

Comment:
It would be great if you add `border` property to a cell during generation the greed. And then add class to it.